Source/Classes/PINSpeedRecorder.h

```diff
@@ -21,8 +21,27 @@ typedef enum : NSUInteger {
                                                   lowQualityQPSThreshold:(float)lowQualityQPSThreshold
                                                  highQualityQPSThreshold:(float)highQualityQPSThreshold;
 
-- (void)addTaskBPS:(float)bytesPerSecond endDate:(NSDate *)endDate;
-- (float)currentBytesPerSecond;
+- (void)processMetrics:(NSURLSessionTaskMetrics *)metrics forTask:(NSURLSessionTask *)task host:(NSString *)host;
```

Comment:
Does it make sense to remove the `host` argument, and instead read it from `task.currentRequest.URL.host`?